index.html

```diff
@@ -1123,16 +1123,36 @@ <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.0.
 
 
 <!-- footer -->
-        <footer>
+        <footer id="responsive-footer">
             <div class="footer-text">
                 <a target="_blank" href="https://pcengines.ch/">
                     https://pcengines.ch/
                 </a>
             </div>
         </footer>
 
-        <a href="#top"><button class="up">UP!</button></a>
+        <a href="#top"><button class="up" id="up-button">UP!</button></a>
         <!-- Go to www.addthis.com/dashboard to customize your tools -->
          <script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-5ad75dbb362a9183"></script>
+
+<script>
+  window.onload = function() {
+      if ( typeof(window.google_jobrunner) === "undefined" ) {
+                function myFunction(x) {
+                    if (x.matches) { // If media query matches
+                        document.getElementById('responsive-footer').style.bottom = "48px";
+                        document.getElementById('up-button').style.bottom = "3.5rem";
+                    }
+                }
+
+                var x = window.matchMedia("(max-width: 980px)")
+                myFunction(x) // Call listener function at run time
+                x.addListener(myFunction) // Attach listener function on state changes
+      } else {
+        console.log("Thank you for not using AddBlock!");
```

Comment:
`AddBlock`? @mscibisz

Reply:
AdBlock* :)

Reply:
@mscibisz is this gonna be fixed or it should be like that?

Reply:
Oh, I thought I just made that typo only in the comment above! Yeah, fixing now.